roles/redpanda_broker/tasks/install-rp-deb.yml

```diff
@@ -1,9 +1,27 @@
 ---
+- name: Check if VM is in GCP
+  uri:
+    url: "http://metadata.google.internal/computeMetadata/v1/instance/id"
+    headers:
+      Metadata-Flavor: "Google"
+    method: GET
+    timeout: 10
+    validate_certs: no
+    status_code: 200
+    return_content: no
+  register: gcp_metadata
+  ignore_errors: yes
+
+- name: Set fact if in GCP
+  set_fact:
+    using_gcp: true
+  when: gcp_metadata.status == 200
```

Comment:
We might be able to get away with looking at ansible_systsem_vendor fact. Atleast on aws, it gets populated automatically to "Amazon EC2". I presume GCP has similar behavior